scripts/src/admin/translations.ts

```diff
@@ -9,6 +9,8 @@ import { ContentType, Offer, OfferContent } from "../models/offers";
 import { OfferTranslation } from "../models/translations";
 import { path } from "../utils/path";
 
+import * as _ from "lodash";
```

Comment:
I really don't see why you need this, please reconsider.

Reply:
done